utils/results/output/securityJobSummary.go

```diff
@@ -187,11 +199,17 @@ func RecordSarifOutput(cmdResults *Results) (err error) {
 		log.Info("Results can be uploaded to Github security tab automatically by upgrading your JFrog subscription.")
```

Comment:
I know it is not a change of yours but this condition doesn't seems right, as it is always evaluated to false:
!extended && !commandsummary.StaticMarkdownConfig.IsExtendedSummary()
(extended is set to 'true' just before checking the condition)

Reply:
Nice, It's a bug. I fixed it